### PR TITLE
add preview note to 2 more Outlook articles

### PR DESCRIPTION
**Pull request by Rick-Kirkham**
Product team requested the note be added to these two articles

### CODE REVIEW

**Comment by jasonjoh** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/add-ins/authentication.md](https://github.com/OfficeDev/outlook-dev-docs/blob/note-about-sso-beta/docs/add-ins/authentication.md) | :white_check_mark:Succeeded | |
[docs/add-ins/implement-sso-in-outlook-add-in.md](https://github.com/OfficeDev/outlook-dev-docs/blob/note-about-sso-beta/docs/add-ins/implement-sso-in-outlook-add-in.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C9%5C5%5Cf49385f6-041e-c905-242e-e0025c631156%5CPullRequest%5C201809052325124778-361%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=%2F8fMjtqTEPDFBotQiSmX1IU1eoCpEgcN5bTVD%2Fh8O14%3D&st=2018-09-05T23%3A21%3A11Z&se=2018-10-06T23%3A26%3A11Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/add-ins/authentication.md [hunk 1]
```diff
@@ -14,7 +14,16 @@ Your Outlook Add-in can access information from anywhere on the internet, whethe
 
 ## Single sign-on access token
 
-Single sign-on access tokens provide a seamless way for your add-in to authenticate and obtain access tokens to call the [Microsoft Graph API](https://developer.microsoft.com/graph/docs/concepts/overview). This capability reduces friction since the user is not required to enter their credentials. Consider using SSO access tokens if your add-in:
+
+Single sign-on access tokens provide a seamless way for your add-in to authenticate and obtain access tokens to call the [Microsoft Graph API](https://developer.microsoft.com/en-us/graph/docs/concepts/overview). This capability reduces friction since the user is not required to enter their credentials.
```
Comment (ElizabethSamuel-MSFT) [inline docs/add-ins/authentication.md]: please remove en-us from the link

File: docs/add-ins/authentication.md [hunk 2]
```diff
@@ -14,7 +14,16 @@ Your Outlook Add-in can access information from anywhere on the internet, whethe
 
 ## Single sign-on access token
 
-Single sign-on access tokens provide a seamless way for your add-in to authenticate and obtain access tokens to call the [Microsoft Graph API](https://developer.microsoft.com/graph/docs/concepts/overview). This capability reduces friction since the user is not required to enter their credentials. Consider using SSO access tokens if your add-in:
+
+Single sign-on access tokens provide a seamless way for your add-in to authenticate and obtain access tokens to call the [Microsoft Graph API](https://developer.microsoft.com/en-us/graph/docs/concepts/overview). This capability reduces friction since the user is not required to enter their credentials. 
+
+> [!NOTE]
+> The Single Sign-on API is currently supported in preview for Word, Excel, Outlook, and PowerPoint. For more information about where the Single Sign-on API is currently supported, see [IdentityAPI requirement sets](https://dev.office.com/reference/add-ins/requirement-sets/identity-api-requirement-sets).
```
Comment (ElizabethSamuel-MSFT) [inline docs/add-ins/authentication.md]: please update dev.office.com links to docs.microsoft.com versions

**Comment by jasonjoh** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/add-ins/authentication.md](https://github.com/OfficeDev/outlook-dev-docs/blob/note-about-sso-beta/docs/add-ins/authentication.md) | :white_check_mark:Succeeded | |
[docs/add-ins/implement-sso-in-outlook-add-in.md](https://github.com/OfficeDev/outlook-dev-docs/blob/note-about-sso-beta/docs/add-ins/implement-sso-in-outlook-add-in.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C9%5C7%5Cf49385f6-041e-c905-242e-e0025c631156%5CPullRequest%5C201809071935109453-361%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=hqowyzyEO8u8vOL4C6O400AMceQrAILzrJXlweFTJ14%3D&st=2018-09-07T19%3A31%3A15Z&se=2018-10-08T19%3A36%3A15Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**